core/modules/rproc/InfileMerger.cc

```diff
@@ -280,11 +169,17 @@ bool InfileMerger::merge(std::shared_ptr<proto::WorkerResponse> response) {
             return false;
         }
     }
-    return _importResponse(response);
+
+    // Nothing to do if size is zero.
+    if (response->result.row_size() != 0) {
```

Comment:
I'm removing _merge in DM-5910 and putting the code here as there's no longer any need to keep it separate with the end of the Mgr class.